### ISSUE TITLE
[Feature request] columns for each team, sort the team and fast swap each slot (like in Kyrios lobby)

### ISSUE THREAD

**Issue by MHRB135** [FAForever/fa#3850]

**Comment by Ejsstiil** [comment]
What if you want 4 teams?

**Comment by Garanas** [comment]
I agree that it is better, but at the moment we don't have the man power to properly tackle the lobby.

**Comment by Hdt80bro** [comment]
I'd be willing to pick it up. I just don't know what we want to have happen in all of the edge cases. In particular, it changes teams being a per player thing to players being a per team thing. How would we handle FFA? More than 2 teams? Survival maps or sandbox mode? Different game layouts per map? How much information would need to be provided by the map maker and what do we do when they don't provide enough? Would optimal balance finally be integrated more fully to show slot positions before the game starts? There's a lot that needs to be decided first.
